python/tank/platform/validation.py

```diff
@@ -208,7 +208,7 @@ def validate_and_return_frameworks(descriptor, environment):
 
                 if min_version_satisfied:
                     found = True
-                    required_fw_instance_names.append(fw_instance_name)
+                    required_fw_instance_names.append((name, fw_instance_name))
```

Comment:
same thought here as above - it's a little confusing that we have name, fw_instance_name etc - what do they mean and how are they different?